src/ShopCertification/Response.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Heureka\ShopCertification;
+
+/**
+ * @author Jakub Chábek <jakub.chabek@heureka.cz>
+ */
+class Response
+{
+
+    /**
+     * @var int
+     */
+    public $code;
+
+    /**
+     * @var string
+     */
+    public $message;
+
+    /**
+     * @var string|null
+     */
+    public $description;
+
+    /**
+     * @param string $json
+     *
+     * @throws JsonException
+     * @throws InvalidResponseException
+     */
+    public function __construct($json)
+    {
+        $response = json_decode($json);
+        if (($response instanceof \stdClass) === false) {
```

Comment:
jasně, ALE on může předat jako JSON pouze `true`, pak `json_decode()` vrátí `true` (protože `true` je validní JSON) a nám to pak chcípne na tom, že přistupujeme ke scaláru jako k objektu...